Animations/Rotate/Clock.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var createAnimation = require('tuxx/Animations').createAnimation;
+var assign = require('object-assign');
+var CommonOpacityProps = require('tuxx/Animations/CommonOpacityProps');
+
```

Comment:
Add comment indicating that this is the default rotate animation similar to what was done on line 6 of 'Scale/Down'.